src/WaveTrack.cpp

```diff
@@ -240,7 +271,32 @@ void WaveTrack::SetPanFromChannelType()
       SetPan( -1.0f );
    else if( mChannel == Track::RightChannel )
       SetPan( 1.0f );
-};
+}
+
+bool WaveTrack::LinkConsistencyCheck()
+{
+   if(!PlayableTrack::LinkConsistencyCheck())
+      return false;
+
+   //no need to check existance of the next track
+   auto next = dynamic_cast<WaveTrack*>(*std::next(GetOwner()->Find(this)));
+   if (next == nullptr)
+   {
+       return false;
```

Comment:
Wait, is it correct to return false whenever a WaveTrack is last, or not followed by another wave track?

No -- you should return false only if the track is mono, link type 0.

Reply:
This doesn't cause a project to fail to load in ProjectFileManager.cpp.  Still it should be fixed, in case the return value needs reuse somewhere else.

Reply:
:/ Thanks for spotting this!